src/content/advanced/gitops/bases/index.md

```diff
@@ -0,0 +1,174 @@
+[create shared template base](#create-shared-template-base)
+and [create versioned base](#create-versioned-base-optional).
+
+
+**IMPORTANT**, CAPx configuration utilizes the
+[App Platform Configuration Levels](/getting-started/app-platform/app-configuration/#levels),
+in the following manner:
+
+- cluster templates provide default configuration via App' `config` field,
+- cluster instances provide custom configuration via App' `userConfig` field, that is overlaid on top of `config`.
+
+See more about this approach [here](https://github.com/giantswarm/rfc/tree/main/merging-configmaps-gitops).
+
+## Choose bases
+
+In order to avoid code duplication, it is advised to utilize the
+[bases and overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays)
+of Kustomize in order to configure the cluster.
+
+## Create shared cluster template base {#create-shared-template-base}
+
+**IMPORTANT:** shared cluster template base should not serve as a standalone base for cluster creation, it is there only to abstract
+App CRs that are common to all clusters versions, and to provide basic configuration for default apps App.
+It is then used as a base to other bases, which provide an overlay with a specific configuration. This is to avoid
+code duplication across bases.
+
+**Bear in mind**, this is not a complete guide of how to create a perfect base, but rather a mere summary of basic
+steps needed to move forward. Hence, instructions here will not always be precise in telling you what to change,
+as this can strongly depend on resources involved, how much of them you would like to include into a base, etc.
+
+We provide a command to create bases for CAPI clusters in an easy way. A base can be created by running:
+
+```nohighlight
+kubectl gs gitops add base --provider capa
+```
+
+This command will create the folder structure and the specific folder for the provider capa, including the template files in the `template` folder. The structure would be:
+
+```nohighlight
+bases
+└── clusters
+    └── capa
+        └── template
+            ├── cluster_config.yaml
+            ├── cluster.yaml
+            ├── default_apps_config.yaml
+            ├── default_apps.yaml
+            └── kustomization.yaml
+```
+
+The current possible values for the providers can be checked in the [command reference]({{< relref "/use-the-api/kubectl-gs/gitops/add-base" >}})
+
+## Create versioned base (optional) {#create-versioned-base-optional}
+
+**IMPORTANT**, versioned cluster template bases use a shared cluster template base and overlay it with a preferably generic configuration for a given cluster version. Versioning comes from the fact that `values.yaml` schema may change over multiple releases, and although minor differences can be handled on the `userConfig` level, it is advised for the bases to follow major `values.yaml` schema versions to avoid confusion.
+
```

Comment:
```suggestion
**IMPORTANT**, versioned cluster template bases use a shared cluster template base and overlay it with a preferably generic configuration for a given cluster version. Versioning comes from the fact that `values.yaml` schema may change over multiple releases, and although minor differences can be handled on the `extraConfig` level, it is advised for the bases to follow major `values.yaml` schema versions to avoid confusion.

```